tock/hours/views.py

```diff
@@ -704,15 +705,16 @@ def get_context_data(self, **kwargs):
         We need to add the reporting period to context,
         as well as users who have not filed timecards,
         so long as they are not new hires or recently departed.
+        We also need to know which organizations have absentee filers.
```

Comment:
Heather, I think you and I both use comments as pseudo-code to map out what we then need to do :-)

Reply:
hey, it's better than writing a `TODO` that I know I'll forget to take out!

Reply:
I love it! I think it makes for good docs!